recipes/libcurl/all/conanfile.py

```diff
@@ -151,6 +153,8 @@ def requirements(self):
             self.requires("brotli/1.0.9")
         if self.options.get_safe("with_zstd"):
             self.requires("zstd/1.4.5")
+        if self.options.get_safe("with_c_ares"):
+            self.requires("c-ares/1.16.1")
```

Comment:
why `get_safe`? This option is never removed. Does this option make sense for all versions currently in CCI?